src/arns.lua

```diff
@@ -1025,31 +989,32 @@ function arns.pruneRecords(currentTimestamp, lastGracePeriodEntryEndTimestamp)
 	return prunedRecords, newGracePeriodRecords
 end
 
---- Prunes auctions that have expired
+--- Prunes returned names that have expired
 --- @param currentTimestamp number The current timestamp
---- @return Auction[] prunedAuctions - the pruned auctions
-function arns.pruneAuctions(currentTimestamp)
-	local prunedAuctions = {}
-	if not NextAuctionsPruneTimestamp or currentTimestamp < NextAuctionsPruneTimestamp then
-		-- No known auctions to prune
-		return prunedAuctions
+--- @return ReturnedName[] prunedReturnedNames - the pruned returned names
+function arns.pruneReturnedNames(currentTimestamp)
+	local prunedReturnedNames = {}
+	if not NextReturnedNamesPruneTimestamp or currentTimestamp < NextReturnedNamesPruneTimestamp then
+		-- No known returned names to prune
+		return prunedReturnedNames
 	end
 
 	local minNextEndTimestamp
-	for name, auction in pairs(arns.getAuctions()) do
-		if auction.endTimestamp <= currentTimestamp then
-			prunedAuctions[name] = arns.removeAuction(name)
+	for name, returnedName in pairs(arns.getReturnedNames()) do
+		local endTimestamp = returnedName.startTimestamp + constants.returnedNamePeriod
+		if endTimestamp <= currentTimestamp then
```

Comment:
For some reason I read this in the inverse of the way my mind sees the relevant time sequences for the affirmative case, i.e. my mind desires:
```
if currentTimestamp >= endTimestamp 
```

Reply:
0c1d597